src/collections/programs/hacktoberfest-2021/index.mdx

```diff
@@ -0,0 +1,165 @@
+---
+title: "Hacktoberfest 2021"
+thumbnail: ./hacktoberfest2021.png
+permalink: "hacktoberfest/2021"
+program: "Hacktoberfest"
+programSlug: "hacktoberfest"
+published: true
+---
+
+import { ProgramsWrapper } from "../Programs.style.js";
+import hacktoberfest2021 from "./hacktoberfest2021.png";
+
+<ProgramsWrapper>
+
+<img class="hacktoberfest-img" src={hacktoberfest2021}/>
+<h2>Participate in Hacktoberfest 2021 with Layer5</h2>
+<div>
+    Hacktoberfest is open to everyone in the global community. Whether you’re
+      a developer, student learning to code, event host, or a company of any
+      size, you can help drive growth of open source and make positive
+      contributions to an ever-growing community. All backgrounds and skill
+      levels are encouraged to complete the challenge. Read more about
+      <a href="https://hacktoberfest.digitalocean.com" target="_blank"> Hacktoberfest</a>.
+</div><br/>
+
+<h4><strong>Hacktoberfest is a celebration open to everyone in our global community.</strong></h4>
+<br/>
+
+<div>
+    While small, the Layer5 community represents the largest collection of
+      service mesh projects and their maintainers in the world. We build
+      projects to provide learning environments, deployment and operational best
+      practices, performance benchmarks, create documentation, share networking
+      opportunities, and more. Our shared commitment to the open source spirit
+      pushes Layer5 projects forward. New members are always welcome.
+</div><br/>
```

Comment:
@XDRAGON2002 please resync with `master` as the formatting and text of the 2022 page has been overhauled.